### ISSUE TITLE
[Bug] Pf2e damage chat message appears in IC tab

### ISSUE THREAD

**Issue by weepingminotaur** [cswendrowski/FoundryVTT-Tabbed-Chatlog#71]
When a creature takes damage or healing, the chat message automatically given by the Pathfinder 2e system appears on the In Character tab, when it should by rights appear on the Rolls tab.

**Comment by Myriade91** [comment]
Yes. It's annoying. I'd like that tab to be only RP-oriented. A fix would be welcome.

**Comment by Charles-Grimlie** [comment]
My solution to this problem was to simply delete the damage-taken.hbs found within PF2e's system folder. I'm certain it's not a feature that'll be missed.

Deleting the file removes the card from the chat log.

**Comment by Myriade91** [comment]
In foundry this works but how can i do that for games on the forge? envoyé : 22 juin 2023 à 19:47de : Charles-Grimlie ***@***.***>à : cswendrowski/FoundryVTT-Tabbed-Chatlog ***@***.***>cc : Myriade91 ***@***.***>, Comment ***@***.***>objet : Re: [cswendrowski/FoundryVTT-Tabbed-Chatlog] [Bug] Pf2e damage chat message appears in IC tab (Issue #71) My solution to this problem was to simply delete the damage-taken.hbs found within PF2e's system folder. I'm certain it's not a feature that'll be missed.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>
